### PR TITLE
cpu9p: implement UnlinkAt and Remove

### PR DESCRIPTION
**Pull request by rminnich**
Signed-off-by: Ronald G. Minnich <rminnich@google.com>

### CODE REVIEW

File: client/cpu9p.go [hunk 1]
```diff
@@ -253,3 +253,21 @@ func (l *cpu9p) Flush() error {
 func (l *cpu9p) Renamed(parent p9.File, newName string) {
 	l.path = filepath.Join(parent.(*cpu9p).path, newName)
 }
+
+// Remove implements p9.File.Remove
+// Open implements p9.File.Open.
```
Comment (brho) [inline client/cpu9p.go]: extra comment about Open?

File: client/cpu9p.go [hunk 2]
```diff
@@ -253,3 +253,21 @@ func (l *cpu9p) Flush() error {
...
+func (l *cpu9p) Remove() error {
+	err := os.Remove(l.path)
+	verbose("Remove(%q): (%v)", l.path, err)
+	return err
+}
+
+// UnlinkAt implements p9.File.UnlinkAt.
+// The flags docs are not very clear, but we
+// always block on the ulink anyway.
```
Comment (brho) [inline client/cpu9p.go]: s/ulink/unlink/ typo

File: client/cpu9p_unix_test.go [hunk 3]
```diff
@@ -197,3 +198,41 @@ func Test9pUnix(t *testing.T) {
...
+	if err := ioutil.WriteFile(f, []byte("hi"), 0666); err != nil {
+		t.Fatalf(`ioutil.WriteFile(%q, "hi", 0666): %v != nil`, f, err)
+	}
+	c := &cpu9p{
+		path: d,
+	}
+
+	err := c.UnlinkAt("a", 0)
+	if err != nil {
+		t.Errorf("Remove: %v != nil", err)
```
Comment (brho) [inline client/cpu9p_unix_test.go]: s/Remove/Unlink/ ?  (here and below)

**Review by brho — APPROVED** [review]
commenting/text nits

File: client/cpu9p.go (hunk 1 above)
Comment (rminnich) [reply client/cpu9p.go]: oh heck. Will fix monday.

**Pull request merged**